addon/utils/ajax.ts

```diff
@@ -1,6 +1,6 @@
-import jQuery from 'jquery';
+import * as jQuery from 'jquery';
```

Comment:
This does type information consumers a favor by allowing `ember-ajax` to work without forcing them to turn on `allowSyntheticDefaultImports` or `esModuleInterop` flags in their `tsconfig.json`. Libraries should generally _not_ enable these flags for this reason

Reply:
Looks like we may need to resolve this in a separate PR. What we have for now is tests that prove consumers may _need_ `"allowSyntheticDefaultImports": true` in their tsconfig in order to use anything that touches `addon/utils/ajax.ts` or the types it emits.